project/Build.scala

```diff
@@ -237,6 +238,11 @@ object BijectionBuild extends Build {
     libraryDependencies += "com.twitter" %% "util-core" % "6.20.0"
   ).dependsOn(bijectionCore % "test->test;compile->compile")
 
+  lazy val bijectionFinagle = module("finagle").settings(
```

Comment:
Should this be a finagle-mysql module? doesn't seem to use finagle core, and similarly a finagle core set of injections wouldn't want the mysql dep. Especially given its experimental